### PR TITLE
[flutter_conductor] Conductor should stop mirroring beta releases to dev

### PR DESCRIPTION
**Pull request by XilaiZhang**
Thanks Christopher for writing up the spec! So I can just be lazy and copy the context link here https://github.com/flutter/flutter/issues/100714

### CODE REVIEW

File: dev/conductor/core/lib/src/next.dart [hunk 1]
```diff
@@ -293,37 +293,29 @@ class NextContext extends Context {
...
 
-            final bool response = await prompt(
-              'Are you ready to publish version ${state.releaseVersion} to $releaseRef?',
-            );
-            if (!response) {
-              stdio.printError('Aborting command.');
-              updateState(state, stdio.logs);
-              return;
-            }
+          final bool response = await prompt('Are you ready to publish this release?');
```
Comment (christopherfujino) [inline dev/conductor/core/lib/src/next.dart:306]: can you revert this message back? The other message has more information.

**Review by christopherfujino — APPROVED** [review]
1 nit about the prompt message, otherwise LGTM

**Review by christopherfujino — APPROVED** [review]
still LGTM

**Comment by XilaiZhang** [comment]
i was cleaning up my branches since i created too many during the flutter roll, and discovered this ancient branch. seems like the tests were failing and it was not merged. is this still applicable and is there anything I can do to help out?

**Comment by christopherfujino** [comment]
> i was cleaning up my branches since i created too many during the flutter roll, and discovered this ancient branch. seems like the tests were failing and it was not merged. is this still applicable and is there anything I can do to help out?

Yeah, we should land this. It's not urgent, but we're publishing unneccessary dev releases.

**Comment by CaseyHillers** [comment]
Any blockers on this I can help with?

**Comment by XilaiZhang** [comment]
aww yes I should find some time to get back on these prs. I am still finishing up on my 3rd version of design doc

**Comment by Hixie** [comment]
What is the status of this PR?

**Comment by XilaiZhang** [comment]
ahhh old memories. I will rebase it after the meeting with christopher

**Pull request merged**